tools/hifiasm/hifiasm.xml

```diff
@@ -354,12 +355,14 @@
             <output name="hic_acontig_graph" file="hifiasm-out-hifi-a.gfa" ftype="gfa1" />
             <output name="hic_balanced_contig_hap1_graph" ftype="gfa1">
                 <assert_contents>
-                    <has_size value="0"/>
```

Comment:
@gallardoalba using has_size 0 is dangerous as we oversee real errors like this one here.

Reply:
Sorry!